Exiled.CustomItems/Events/PlayerHandler.cs

```diff
@@ -19,36 +19,26 @@ namespace Exiled.CustomItems.Events
     /// </summary>
     internal sealed class PlayerHandler
     {
-        /// <inheritdoc cref="ChangingRoleEventArgs"/>
-        public void OnChangingRole(ChangingRoleEventArgs ev)
+        /// <inheritdoc cref="ChangingItemEventArgs"/>
+        public void OnChangingItem(ChangingItemEventArgs ev)
         {
-            if (ev.NewRole == RoleTypeId.Spectator)
+            if (ev.IsAllowed)
+                return;
```

Comment:
Is it supposed to be like this?

Reply:
i mean if we got an ChangedItemEventArgs it's will not need this